tests/test.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+import requests
+import scipy.io.wavfile
+import numpy
+
+
```

Comment:
We should add tests (however trivial they might be) for each endpoint.

Reply:
I think we are doing that already? I have a for loop on line 13 that checks every endpoint

Reply:
Right, but I was thinking about the swagger endpoint `/` and the metadata endpoint `/model/metadata`.

Reply:
Ohhh, I understand. Currently none of the other integration tests are testing the `/model/metadata` endpoint, do you think we need to add this to all of the models?

Reply:
yes please. as trivial as the endpoint implementations might be, it'd be good to get as close to 100% coverage as possible

Reply:
For the swagger, maybe just check the HTTP code and that the response is JSON? I don't really know how frequently the JSON response changes and how painful it'd be to keep the tests up to date. Your call.